### DISCUSSION TITLE
Am I the only one who cannot scale to "1.1"?

### DISCUSSION THREAD

**Discussion by rempas** [hyprwm/Hyprland#5492] [Q&A] [answered]
When I try to scale to **1.1**, I get an error message (in the top of the screen) that says:

`Invalid scale passed to monitor DP-1, failed to find a clean divisor. Suggested nearest scale: 1.200000`

The thing is, **1.2** scales too much, and I want something probably close to **1.1** or **1.15**. Am I the only one who has this problem?

**Comment by thejch** [comment, marked as answer]
debug:disable_scale_checks = true

**Reply by rempas** [reply]
Thank you, it works! I also like how it properly scales every app, unlink Gnome!

Two questions:

1. The text appears to be "pixelated" and "wrong" under Librewolf. Any similar experiences for you?

2. Why isn't this the default? Are there any problems that it "checks"?

**Reply by vaxerski** [reply]
re 1) make sure librewolf is running under wayland native. If it is and it's still pixelated, that's precisely your answer to 2.

**Reply by rempas** [reply]
@vaxerski Thank you! Yes, it does run under Wayland! Actually, after writing this, I noticed that every text is shown like this. It is as if it's not properly rendered and antialised.

Thank you for your reply and your work!
